config/log4j2-rolling.yaml

```
@@ -0,0 +1,105 @@
+Configuration:
+  status: WARN
+  Appenders:
+    RollingFile:
+      - name: main
+        fileName: ${ksql.log.dir}/ksql.log
+        filePattern: ${ksql.log.dir}/ksql-%d{yyyy-MM-dd}.log.gz
+        PatternLayout:
+          pattern: "[%d] %p %m (%c:%L)%n"
+        Policies:
+          SizeBasedTriggeringPolicy:
+            size: 10MB
+        DefaultRolloverStrategy:
+          max: 5
+
+      - name: streams
+        fileName: ${ksql.log.dir}/ksql-streams.log
+        filePattern: ${ksql.log.dir}/ksql-streams-%d{yyyy-MM-dd}.log.gz
```

Optional (Old Pattern)
```suggestion
        filePattern: ${ksql.log.dir}/ksql-streams-%i.log
```